CONTRIBUTING.md

```diff
@@ -1,126 +1,241 @@
 # Contributing to Vald
 
-Vald is an open source project.
+Thank you for your interest in Vald, and thank you for investing your time in contributing to Vald!
 
-We appreciate your help!
+Please read our [Code Of Conduct](https://github.com/vdaas/vald/blob/main/CODE_OF_CONDUCT.md) to keep our community approachable and respectable.
 
-## Table of Contents
+In this guide, you will get an idea of how to contribute to Vald.
 
-- [Contributing Issue](#Contributing-Issue)
-  - [Bug Report](#Bug-Report)
-  - [Proposal](#Proposal)
-  - [Feature Request](#Feature-Request)
-  - [Security Issue Report](#Security-Issue-Report)
-- [Contributing Source Code](#Contributing-Source-Code)
-  - [Before contributing source code](#Before-contributing-source-code)
-  - [How to contribute source code](#How-to-contribute-source-code)
-  - [Branch naming convention](#Branch-naming-convention)
+If you are not a developer, don't worry, some contributions don't require writing a single line of code.
 
-## Contributing Issue
+## Contributions
 
-We use [Github Issues](https://github.com/vdaas/vald/issues) to track issues within this repository.
-If you can determine the problem you are facing is a bug or issue, you can easily submit the issues.
+Please read the [README](https://github.com/vdaas/vald/blob/main/README.md) to get an overview of Vald.
 
-If you are unsure whether you have found a bug or security-related issues, please consider asking in the [Vald Slack](https://join.slack.com/t/vald-community/shared_invite/zt-db2ky9o4-R_9p2sVp8xRwztVa8gfnPA) first.
-If the behavior you are seeing is confirmed as a bug or issue, it can easily be re-raised in the issue tracker.
+We welcome you to contribute to Vald to make Vald better.
+We accept the following types of contributions:
 
-### Bug Report
+- Issue
+  - Bug report
+  - Feature request
+  - Proposal
+  - Security issue report
+- Pull request
 
-A bug is a demonstrable problem that produces an incorrect result or behaves in unintended ways.<br>
+Please also feel free to ask anything on [Vald Slack channel](https://join.slack.com/t/vald-community/shared_invite/zt-db2ky9o4-R_9p2sVp8xRwztVa8gfnPA) :)
 
-Bug reports are helpful for developers who maintain the Vald project.<br>
-A good bug report should not leave others needing to constantly ask you for more information.<br>
-Please try to write as detailed as possible in your bug report.
+## Issue contribution
 
-When filing an issue, make sure to answer these five questions:
+Please find [here](./docs/contributing/issue.md) for more details about how to contribute to the issue.
```

Comment:
Too short.
At least, we have to introduce the `issue contribution` for the Vald project.